### DISCUSSION TITLE
How can I restore the backup database?

### DISCUSSION THREAD

**Discussion by tainv-93** [dani-garcia/vaultwarden#3327] [Q&A] [answered]
I used scp to copy sqlite database to mount the volume of the container, but It's not working when I restart the container

**Comment by tainv-93** [comment]
Can I remove two file: `db.sqlite3-shm  db.sqlite3-wal`?

**Comment by jjlin** [comment, marked as answer]
See https://github.com/dani-garcia/vaultwarden/wiki/Backing-up-your-vault
